features/multi-column.yml.dist

```diff
@@ -73,6 +73,19 @@ compat_features:
   #   safari_ios: "10"
   - css.properties.column-gap.multicol_context
 
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "79"
+  #   firefox: "50"
+  #   firefox_android: "50"
+  #   safari: "9"
+  #   safari_ios: "9"
+  - css.properties.column-width.auto
```

Comment:
I'm confused here. The browser versions for the `auto` value on MDN doesn't match what we have here:
![image](https://github.com/user-attachments/assets/f2828ef5-6968-4158-8122-2a1fd523a551)

Checking BCD confirms what's on MDN:
![image](https://github.com/user-attachments/assets/6956e185-1cb4-4c77-895d-afd5461ad6cc)

Any idea why our code returns these older browser versions for this key?

Reply:
That's right. `compute-baseline` checks ancestor compat features (we turned this on in https://github.com/web-platform-dx/web-features/pull/1485). This is an oddity of BCD's data model. It can't directly express the idea that a feature is, for example, behind a prefix when some parent of that feature is behind a parent.